Clients/GUI/wwwroot/css/site.css

```diff
@@ -182,24 +131,28 @@ app {
     }
 }
 
-.newsbar {
-    width: 120px;
+.currencies {
```

Comment:
Also may be finally move these custom styles also from this file away?

Reply:
fixed